### ISSUE TITLE
Rename "Move" button to "Rename" on GCpedia toolbar

### ISSUE THREAD

**Issue by christinaganotakis** [gctools-outilsgc/gcpedia#33]
A user from the Translation Bureau noticed there was an error in a translation in our GCpedia page toolbar. The "Move" button should be changed to "Rename" for translation and accessibility purposes.

**Comment by christinaganotakis** [comment]
![help desk 2](https://cloud.githubusercontent.com/assets/24302797/22569619/44247594-e966-11e6-9b56-4b89f795a2e8.PNG)
![help desk 3](https://cloud.githubusercontent.com/assets/24302797/22569626/48f855f4-e966-11e6-91e2-a2ed15436cc2.PNG)

**Comment by pabrams** [comment]
The functionality seems more like a move than a rename, to me.  Traditionally, a rename operation is a simple kind of move, and the functionality provided can definitely do more than just rename, so to me it's the French wording that should change, unless there's some complicated reasoning behind that translation.

**Comment by ToferC** [comment]
Agree that French word should change. Move is standard wiki parlance.

**Comment by pabrams** [comment]
I suspect that in French, the standard wiki parlance is "renommer", in which case the translation is correct, and this Issue could be closed since no change would be needed at all.  Take a look at the following two pages, which seem like they support what I'm saying:
https://en.wikipedia.org/wiki/Wikipedia:Moving_a_page
https://fr.wikipedia.org/wiki/Aide:Comment_renommer_une_page

**Comment by ToferC** [comment]
Thanks for the follow-up @pabrams.  Closing the issue.
